docs/conversation.md

```diff
@@ -100,6 +100,7 @@
 | `status`                | string  | v4    |         | Optional: Only available for one-to-one conversations and when  `includeStatus=true` is set                                                                                                                                                                                                                                                                                                       |
 | `statusIcon`            | string  | v4    |         | Optional: Only available for one-to-one conversations and when  `includeStatus=true` is set                                                                                                                                                                                                                                                                                                       |
 | `statusMessage`         | string  | v4    |         | Optional: Only available for one-to-one conversations and when  `includeStatus=true` is set                                                                                                                                                                                                                                                                                                       |
+| `statusClearAt`         | int     | v4    |         | Optional: Only available for one-to-one conversations and when  `includeStatus=true` is set                                                                                                                                                                                                                                                                                                       |
```

Comment:
I think that would be good talk that this value can be null. Isn't only integer every time.